app/controller/crackme.go

```diff
@@ -1,22 +1,23 @@
 package controller
 
 import (
-    "github.com/xusheng6/crackmes.one/app/model"
```

Comment:
Try not to change the spacing (tabs v.s. space), and the order of the modules unless necessary